test/e2e/nodemaintenance_test.go

```diff
@@ -30,6 +32,30 @@ var (
 	podLabel             = map[string]string{"test": "drain"}
 )
 
+const (
+	scriptShowCRD = `./kubevirtci/cluster-up/kubectl.sh get -o yaml crd nodemaintenances.kubevirt.io`
```

Comment:
This is `Go`, is there not a programatic way to obtain the yaml of a CRD from the server?

Reply:
kubectl seems to be running this function in order to do the kubectl get command;
what i did't find so far is how they handle the -o json option.

repo: https://github.com/kubernetes/kubernetes.git
file: ./staging/src/k8s.io/kubectl/pkg/cmd/get/get.go                                                                                                                             
```
// Run performs the get operation.
// TODO: remove the need to pass these arguments, like other commands.
func (o *GetOptions) Run(f cmdutil.Factory, cmd *cobra.Command, args []string) error {
	if len(o.Raw) > 0 {
		restClient, err := f.RESTClient()
		if err != nil {
			return err
		}
		return rawhttp.RawGet(restClient, o.IOStreams, o.Raw)
	}
	if o.Watch || o.WatchOnly {
		return o.watch(f, cmd, args)
	}

	chunkSize := o.ChunkSize
	if o.Sort {
		// TODO(juanvallejo): in the future, we could have the client use chunking
		// to gather all results, then sort them all at the end to reduce server load.
		chunkSize = 0
	}

	r := f.NewBuilder().
		Unstructured().
		NamespaceParam(o.Namespace).DefaultNamespace().AllNamespaces(o.AllNamespaces).
		FilenameParam(o.ExplicitNamespace, &o.FilenameOptions).
		LabelSelectorParam(o.LabelSelector).
		FieldSelectorParam(o.FieldSelector).
		ExportParam(o.Export).
		RequestChunksOf(chunkSize).
		ResourceTypeOrNameArgs(true, args...).
		ContinueOnError().
		Latest().
		Flatten().
		TransformRequests(o.transformRequests).
		Do()


```

Reply:
fyi, the canonical location is

    https://github.com/kubernetes/kubectl/blob/HEAD/pkg/cmd/get/get.go

Why not import the `get` package in the same way that we consume the `drain` package, and do what their test code does?

    https://github.com/kubernetes/kubectl/blob/HEAD/pkg/cmd/get/get_test.go#L144

Reply:
This one file is requiring a whole lot of dependencies - and it will be quite hard to get them working all togather.